src/vcSceneLayer.cpp

```diff
@@ -353,6 +353,8 @@ udResult vcSceneLayer_LoadFeatureData(const vcSceneLayer *pSceneLayer, vcSceneLa
 
     pNode->pFeatureData[i].pGeometryLayout = udAllocType(vcVertexLayoutTypes, pNode->pFeatureData[i].geometryLayoutCount, udAF_Zero);
     UD_ERROR_NULL(pNode->pFeatureData[i].pGeometryLayout, udR_MemoryAllocationFailure);
+    pNode->pFeatureData[i].pAttributeOffset = udAllocType(uint64_t, pNode->pFeatureData[i].geometryLayoutCount, udAF_Zero);
```

Comment:
This is unrelated work for the SLPK things?

Reply:
Yes, I'm sort of confused as to how that got into this PR, I thought it had been merged already, I'm guessing it was in master at some point and then I started working on top of that branch... Then maybe pete dumped the attribute offset change? He mentioned to me on Tuesday that the other change I made related to that was the actual fix and this change didn't contribute but nonetheless was correct and should be done